segmentation_models_pytorch/fpn/model.py

```diff
@@ -49,6 +52,7 @@ def __init__(
             final_channels=classes,
             dropout=dropout,
             final_upsampling=final_upsampling,
+            merge_policy=check_merge_policy(decoder_merge_policy)
```

Comment:
Lets make it only inside decoder ;)
without utility function, just 
```python
    if merge_policy not in ['add', 'cat']:
        raise ValueError("`merge_policy` must be one of: ['add', 'cat'], got {}".format(merge_policy)
```
And without `f` string formatting (in this project it is not used, it is better to be cosistent)

Reply:
Fixed.